### DISCUSSION TITLE
Use of `Any{SomeKey} >> SomeOtherKey` prevents key repeat, breaks normal chord behavior.

### DISCUSSION THREAD

**Discussion by ghost** [houmain/keymapper#204] [Q&A]
Key repeat stops working entirely, and as a consequnece chords are also registered only after key release (meaning you have to press and release them very deliberately). So it will basically break your keyboard.

I can kind of see the logic behind this, since SomeKey is itself an element of Any and as such waits to see whether it iself will end up as a modifier for Somekey, i.e. Any{SomeKey{SomeKey}}. Of course in practice this would never happen, unless there there are two exactly identical keys on the board and the user for whatever reason wants to use both of them in the same chord, so fall intents and purposes this scenario is impossible.

It may therefore be a good idea to at least throw a warning when something like this added to the config.

**Comment by ristomatti** [comment]
Are you using mouse wheel mappings by any chance? I remember at some point using them just once triggered a bug, but it got fixed IIRC.

Can you provide a minimal as possible config that triggers the issue? (Or the very least the full config)

**Reply by ghost** [reply]
Sorry for the late reply, but the answer is no. I've tested with just this one config. I think the behavior is expected no?

**Comment by houmain** [comment]
> I can kind of see the logic behind this, since SomeKey is itself an element of Any and as such waits

I don't think it has anything to do with SomeKey also being an element of Any. With this configuration `Any{SomeKey} >> SomeOtherKey` it simply waits after any key whether a SomeKey follows or not.

**Reply by ghost** [reply]
The "issue" is that it waits until all keys are released.
